### PR TITLE
revert the geneset edit UI to not display groups controls

### PR DESCRIPTION
**Pull request by siosonel**
## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
- [x] Tests: added and/or passed unit and integration tests, or N/A
- [x] Todos: commented or documented, or N/A
- [x] Notable Changes: updated release.txt, prefixed a commit message with "fix:" or "feat:", added to an internal tracking document, or N/A

### CODE REVIEW

File: client/dom/genesetEdit.ts [hunk 1]
```diff
@@ -19,39 +19,38 @@ type Gene = { name: string }
...
 	holder: any
 	genome: any
-	geneList?: Gene[]
 	mode?: string
 	callback: (CallbackArg) => void
 	vocabApi: any
-	groups?: { name: string; lst: Gene[]; type?: 'hierCluster' }[]
-	showGroup: boolean
+
+	geneList: {
```
Comment (xzhou82) [inline client/dom/genesetEdit.ts:31]: should be optional

File: client/dom/genesetEdit.ts [hunk 2]
```diff
@@ -19,39 +19,38 @@ type Gene = { name: string }
...
 	callback: (CallbackArg) => void
 	vocabApi: any
-	groups?: { name: string; lst: Gene[]; type?: 'hierCluster' }[]
-	showGroup: boolean
+
+	geneList?: {
+		name: string
+	}[]
+
+	titleText: string
```
Comment (xzhou82) [inline client/dom/genesetEdit.ts:35]: also optional

File: client/dom/genesetEdit.ts [hunk 3]
```diff
@@ -19,39 +19,38 @@ type Gene = { name: string }
...
+
 	backBtn: {
```
Comment (xzhou82) [inline client/dom/genesetEdit.ts:37]: optional

**Comment by siosonel** [comment]
> I liked better the other way, that was not showing the groups in the main menu. I felt it was more focused in the genes than in the groups, this way to me is more confusing. But this is just my opinion. Please disregard. I am just saying it here because I am a reviewer.

@airenzp My last commit combines the group selector/name input in the same menu as the geneset edit UI, while still separating the group handling/logic out of the geneset edit code.

**Comment by xzhou82** [comment]
top variably expressed gene btn is no longer there

**Review by xzhou82 — APPROVED** [review]

**Pull request merged**